tutorials/W1D2_ComparingTasks/W1D2_Tutorial3.ipynb

```diff
@@ -250,16 +263,90 @@
+    "\n",
+    "class PlayVideo(IFrame):\n",
+    "  def __init__(self, id, source, page=1, width=400, height=300, **kwargs):\n",
+    "    self.id = id\n",
+    "    if source == 'Bilibili':\n",
+    "      src = f'https://player.bilibili.com/player.html?bvid={id}&page={page}'\n",
+    "    elif source == 'Osf':\n",
+    "      src = f'https://mfr.ca-1.osf.io/render?url=https://osf.io/download/{id}/?direct%26mode=render'\n",
+    "    super(PlayVideo, self).__init__(src, width, height, **kwargs)\n",
+    "\n",
+    "\n",
+    "def display_videos(video_ids, W=400, H=300, fs=1):\n",
+    "  tab_contents = []\n",
+    "  for i, video_id in enumerate(video_ids):\n",
+    "    out = widgets.Output()\n",
+    "    with out:\n",
+    "      if video_ids[i][0] == 'Youtube':\n",
+    "        video = YouTubeVideo(id=video_ids[i][1], width=W,\n",
+    "                             height=H, fs=fs, rel=0)\n",
+    "        print(f'Video available at https://youtube.com/watch?v={video.id}')\n",
+    "      else:\n",
+    "        video = PlayVideo(id=video_ids[i][1], source=video_ids[i][0], width=W,\n",
+    "                          height=H, fs=fs, autoplay=False)\n",
+    "        if video_ids[i][0] == 'Bilibili':\n",
+    "          print(f'Video available at https://www.bilibili.com/video/{video.id}')\n",
+    "        elif video_ids[i][0] == 'Osf':\n",
+    "          print(f'Video available at https://osf.io/{video.id}')\n",
+    "      display(video)\n",
+    "    tab_contents.append(out)\n",
+    "  return tab_contents\n",
+    "\n",
+    "\n",
+    "video_ids = [('Youtube', 'YDOhHzVUpGU'), ('Bilibili', 'BV1bbVjzyE34')]\n",
+    "tab_contents = display_videos(video_ids, W=854, H=480)\n",
+    "tabs = widgets.Tab()\n",
+    "tabs.children = tab_contents\n",
+    "for i in range(len(tab_contents)):\n",
+    "  tabs.set_title(i, video_ids[i][0])\n",
+    "display(tabs)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# @title Submit your feedback\n",
+    "content_review(f\"{feedback_prefix}_pomdps\")"
+   ]
+  },
   {
    "cell_type": "markdown",
    "metadata": {
     "execution": {}
    },
    "source": [
     "---\n",
-    "# Section 1: Reinforcement learning across temporal scales\n",
+    "# Section 2: Reinforcement learning across temporal scales\n",
     "\n",
-    "In this section we will review the basics of reinforcement learning as well as we will introduce meta-reinforcement learning (meta-RL). For that, we will study how \"learning to learn\" is implemented in the context of a binary bandit task and how learning is affected by changing temporal scales."
+    "In this section we will review the basics of reinforcement learning as well as introducing meta-reinforcement learning (Meta RL). For that, we will study how \"learning to learn\" is implemented in the context of a binary bandit task and how learning is affected by changing temporal scales."
```

Comment:
Lets be consistent with Meta-RL or Meta RL

Reply:
Thanks for pointing this out. I've amended the notebook so it is consistent throughout.